src/classes/STG_PanelPaymentMapping_CTRL.cls

```diff
@@ -342,12 +347,21 @@ public with sharing class STG_PanelPaymentMapping_CTRL extends STG_Panel {
                     pfm2.Id = pfm.Id;
                     pfm2.npe01__Opportunity_Field__c = strOppField;
                     pfm2.npe01__Payment_Field__c = strPmtField;
-                    listPMTFieldMapping.add(pfm2);                                      
-                }                
-            } 
-            return listPMTFieldMapping;           
+                    listPMTFieldMapping.add(pfm2);
+                }
+            }
+            return listPMTFieldMapping;
         }
-        private set;     
+        private set;
+    }
+
+    /***
+    * @description Verifies the user has Delete permissions for Opportunity naming settings.
```

Comment:
@RadSFDeveloper , I will change to isAdmin as we need to check all three permissions in this WI. Since this is used within the controller, can I make it as private Boolean isAdmin?

Reply:
or I can just check this condition STG_Panel.runningUserIsAdmin() in the controller instead of separate method.